### ISSUE TITLE
Schema onboarding flow runs on existing users' new instances

### ISSUE THREAD

**Issue by kakoga** [zesty-io/manager-ui#187]
**Describe the bug**
The schema onboarding flow runs not only for new users but also for existing users who have reset a blueprint or created a new instance.

**To Reproduce**
1. In accounts, either create a new instance or reset a blueprint on an existing instance
2. Open legacy manager
3. Get launched into Schema new user onboarding flow

**Expected behavior**
The Schema new user onboarding flow should only run for new users. It shouldn't run for existing users who are creating new instances or for users who are resetting blueprints.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac Catalina
 - Browser Chrome
 - Version 85.0.4183.121 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
n/a

**Additional context**
Copied from @nibblebot ticket.
> Once the instance creation logic is in place, we should bounce "first time users" (first session after creating the instance) to /schema/start if they visit /schema

**Comment by shrunyan** [comment]
Marking as wontfix and closing. I think we are fine having this flow trigger any instance in a "new" state, whether that is on initial creation or reset. As this is a flow which can easily be left if desired.
